### PR TITLE
Verbose mode update

### PR DESCRIPTION
**Pull request by Alexsandruss**
Changes:

- Create named logger for sklearnex
- Update logging workflow in daal4py/sklearnex estimators
- Add verbose mode to TSNE
- Update for verbose doc page
- Unify `_onedal_*_supported` methods in kNN/SVM

Solves https://github.com/intel/scikit-learn-intelex/issues/635

### CODE REVIEW

**Comment by napetrov** [comment]
@Alexsandruss  - suggest to also update example https://github.com/intel/scikit-learn-intelex/blob/master/examples/sklearnex/verbose_mode.py

**Comment by Alexsandruss** [comment]
@napetrov, I updated verbose mode example to use new logger instead of modifying env.variable inside script. Do you think env.variable still should be mentioned in example?

**Comment by napetrov** [comment]
It's fine to go without env.var in example, but keep it in documentation

**Review by inteldimitrius — APPROVED** [review]
Changes look good to me.

File: doc/sources/verbose.rst [hunk 1]
```diff
@@ -35,14 +35,20 @@ To enable verbose mode, set the ``SKLEARNEX_VERBOSE`` environment variable as sh
 
      set SKLEARNEX_VERBOSE=INFO
 
+Alternatively, get |intelex| logger and set its logging level in Python code::
```
Comment (alexandraepan) [inline doc/sources/verbose.rst:38]:
```suggestion
Alternatively, get |intelex| logger and set its logging level in the Python code::
```

**Comment by md-shafiul-alam** [comment]
PR looks good to me

**Pull request merged**